server/routes/referrals/referralFormPresenter.ts

```diff
@@ -324,17 +338,17 @@ class TaskValues {
     ]
   }
 
-  get relevantSentence(): DraftReferralValues {
-    return [this.referral.relevantSentenceId]
-  }
-
   get cohortServiceCategories(): DraftReferralValues {
-    if (this.referral.serviceCategoryIds === null) {
+    if (this.referral.serviceCategoryIds === null || this.referral.serviceCategoryIds.length === 0) {
```

Comment:
even though we expect serviceCategoryIds to be null - the API appears to return an emptyList

Reply:
that sounds like something that'd be great in a contract

Reply:
great to have a test added for this 👏

Reply:
I did add a test :-) it's in referralFormPresenter.test.ts
it tests if it's both null or empty

Reply:
yeah, sorry, I was happy that it was added :)